pallets/staking/src/lib.rs

```diff
@@ -0,0 +1,261 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_election_provider_support::{
+	ElectionDataProvider, ElectionProvider, ElectionProviderBase, SortedListProvider, Supports,
+};
+use frame_support::defensive;
+use frame_support::{traits::Defensive, RuntimeDebug};
+#[cfg(feature = "std")]
+pub use pallet_staking_substrate::GenesisConfig;
+#[cfg(feature = "std")]
+pub use pallet_staking_substrate::TestBenchmarkingConfig;
+use pallet_staking_substrate::ValidatorPrefs;
+pub use pallet_staking_substrate::{
+	ActiveEra, ActiveEraInfo, Config, ErasStartSessionIndex, Error, Event, ForceEra, Forcing,
+	Pallet, RewardDestination, UseValidatorsMap,
+};
+pub use pallet_staking_substrate::{
+	__InherentHiddenInstance, __substrate_call_check, __substrate_event_check,
+	__substrate_genesis_config_check, tt_default_parts, tt_error_token,
+};
```

Comment:
```suggestion
pub use pallet_staking_substrate as pallet;
```
Then when you construct the runtime

```rust
construct_runtime!(
	pub enum Runtime
	where
		Block = Block,
		NodeBlock = opaque::Block,
		UncheckedExtrinsic = UncheckedExtrinsic
	{
		System: frame_system,
		Timestamp: pallet_timestamp,
		Balances: pallet_balances,
		Staking: pallet_staking::pallet,
		Session: pallet_session,
	}
);
```

Feels less gross than re-exporting macro implementation details.